### DISCUSSION TITLE
【注意】最近一批以太检测试纸可能不正常！！

### DISCUSSION THREAD

**Discussion by xing-qiong** [share121/inter-knot#3199] [其他]
![(低分数)试纸照片](https://github.com/user-attachments/assets/166d6af0-9af7-4451-b52c-af3098c08d67)
最近在绳网上突然冒出一大批以太试纸检验结果分数很高的分享贴，有消息称最近生产的一批试纸可能出了问题，请大家不要看到自己的检测结果高就贸然进入空洞，空洞内环境是非常危险的。
请大家切记：试纸所测结果会受很多因素影响，例如心情不好，近期感冒或者过度劳累等等都会影响到测试结果。最准确的方式还是去医院或者空洞调查协会进行准确检验。

PS：绝对不是楼主检测结果很低才发贴的哦

**Comment by sr-BEEP** [comment]
这样吗，呜啊我之前测了一下发现以太适应超强，刚刚还在和朋友商量着明天偷偷去空洞看看呢，现在不敢想去了会怎样

**Reply by xing-qiong** [reply]
空洞还是很危险的，里面空间很混乱，说不定只踏进空洞几米远空间就会扭曲从而丢失回去的路，要去的话记得带着萝卜。

诶<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >，我这里刚好有一些小型空洞的萝卜数据哦

**Comment by KUYUE404** [comment]
wok，我以为自己是天选空洞打灰圣体呢，没想到是试纸出了问题<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
